src/MissionManager/MissionManager.cc

```diff
@@ -46,7 +46,7 @@ void MissionManager::writeArduPilotGuidedMissionItem(const QGeoCoordinate& gotoC
     mavlink_message_t       messageOut;
     mavlink_mission_item_t  missionItem;
```

Comment:
You could also drop the memset entirely. 
```suggestion
    mavlink_mission_item_t  missionItem{};
```

Reply:
I think that will work. Not quite sure though. We've had problems with some of the newer c++ stuff working on all platforms.